website updates (#7)
bigscience-workshop/bigscience-workshop.github.io: en/pages/faq.md

## Will several models and/or dataset be created - will there be distilled or smaller versions of the model?

> All these questions are currently discussed and not answered. Though given the size targeted for the model it is already pretty clear that only on single main large model will be trained. Distilling or sharing smaller version of the model could be investigated, keeping in mind that the general research goal of the workshop/project. 

Reviewer: ```suggestion
All these questions are currently discussed and not answered. Though given the size targeted for the model it is already pretty clear that only on single main large model will be trained. Distilling or sharing smaller version of the model could be investigated if we have enough compute remaining, keeping in mind the general research goal of the workshop/project. 
```

## How is this energy efficient? Why not smaller models?

We're exploring different options and have dedicated working groups that look into carbon footprint and the overall (social) impact of the project.

## What would happen if some participants don’t finish the task they promised to do?

> Sometimes life gets hectic. It's not a problem if you're not able to finish a task that you promised to do, but please keep in mind that this is a collaborative effort. Make sure that you communicate issues timely with chairs and members, as it may impact their work and overall project outcome.

Reviewer: ```suggestion
Sometimes life gets hectic. It's not a problem if you're not able to finish a task that you promised to do, but please keep in mind that this is a collaborative effort. Make sure that you communicate issues timely with chairs and members, as it may impact their work and overall project outcome. In general, BigScience works following a "graceful" degradation paradigm so that if one working group is not able to fully complete his goals, a simpler version of this working group work should be able to be implemented instead to guarantee the overall completion of the project.
```

## How is the access to the compute facilities/providers handled?

If you need access to compute that goes beyond freely available compute, such as Google Colab, please get in touch with your working group chairs.

## How is the decision making process handled?

> BigScience is an open, distributed and collaborative project with specific working groups that are dedicated to different research areas of large language modeling. Decisions are aligned with the goals and principles of BigScience to conduct open fundamental research, and are made within working groups by chairs and members as well as across working groups that depend on each other.

Reviewer: ```suggestion
BigScience is an open, distributed and collaborative project with specific working groups that are dedicated to different research areas of large language modeling.
Working groups conduct a decision process based on discussion of all possible options and comparison if necessary. In most cases, the process should be to discuss all options and try to reach unanimity around one option. If a decision can not be taken at unanimity and the possible options appear not possible to compare, the working group chair can make a decision for the working group project to move forward
Decisions are aligned with the goals and principles of BigScience to conduct open fundamental research.
The decision process is generally handled by the chairs of each working groups for decision taken inside a single working group and by gathering all the chairs for cross-working groups decisions.
```

## Which languages will be in the dataset?

> The project starts around 8 broadly defined language groups, namely: Arabic, Bantu languages, Chinese, English, French, Hindi and Urdu, Portuguese, and Spanish. However, there are still a number of questions as to how we conceptualize the notion of languages, especially in a way that is less focused on European ones, as this definition has bearing not only on the way resources are allocated during the dataset creation, but also on tokenization, modeling, and even evaluation choices.

Reviewer: ```suggestion
The project starts around 8 broadly defined language groups, namely: Arabic, Bantu languages, Chinese, English, French, Hindi and Urdu, Portuguese, and Spanish which reflect the most spoken languages on earth. However, there are still a number of questions as to how we conceptualize the notion of languages, especially in a way that is less focused on European ones, as this definition has bearing not only on the way resources are allocated during the dataset creation, but also on tokenization, modeling, and even evaluation choices.
More generally, the list of language is not yet fixed and will depend on the community involvement for each language group. The focus on the BigScience project is on obtaining quality sources for each finally selected languages rather than seeking to support a maximal number of languages. The present FAQ is not frequently updated so all the recent developments on the selection of languages should be asked directly to the "Dataset" working groups and in particular the "Dataset - Governance" working group.
```

## Who is behind this initiative?

You can find a brief history and the founding members of the BigScience project [here](https://bigscience.huggingface.co/en/#!pages/description.md).

## Could the model, whose creation is proposed during the shared task, be a multimodal model?

> The language model is trained on text, however, it's still possible to come up with new ideas and create new working groups within the project.

Reviewer: ```suggestion
For computation purposes and since this project already has many experimental aspects (including the organisation it-self), the model will likely be trained on text only (including possibly non-natural-language like code).
```
